dataactbroker/handlers/jobHandler.py

```diff
@@ -52,27 +53,60 @@ def loadSubmitParams(cls,requestDict):
             if requestDict.exists(key):
                 if(key == "reporting_period_start_date" or key == "reporting_period_end_date"):
                     reportDate = requestDict.getValue(key)
-                    # Check if this is specified by quarter or month
-                    if(reportDate[0] == "Q"):
-                        if key == "reporting_period_start_date":
-                            isStart = True
-                        else:
-                            isStart = False
-                        reportDate = "".join([cls.quarterToMonth(reportDate[0:2], isStart),reportDate[2:]])
 
                     # Create a date object from formatted string, assuming "MM/YYYY"
                     try:
                         submissionData[cls.metaDataFieldMap[key]] = JobHandler.createDate(reportDate)
                     except ValueError as e:
                         # Bad value, must be MM/YYYY
-                        raise ResponseException("Date must be provided as MM/YYYY or Q#/YYYY",StatusCode.CLIENT_ERROR,ValueError)
+                        raise ResponseException("Date must be provided as MM/YYYY",StatusCode.CLIENT_ERROR,ValueError)
                 else:
                     submissionData[cls.metaDataFieldMap[key]] = requestDict.getValue(key)
             else:
                 if not existingSubmission:
                     raise ResponseException(key + " is required",StatusCode.CLIENT_ERROR,ValueError)
         return submissionData, existingSubmissionId
 
+    def getStartDate(self, submission):
+        """ Return formatted start date """
+        if submission.is_quarter_format:
+            quarter = self.monthToQuarter(submission.reporting_start_date.month, True)
+            year = submission.reporting_start_date.year
+            if quarter == "Q1":
```

Comment:
Wanted to check on this because @kevinli-work was asking me about this yesterday (i.e., which fiscal year Q1 months belong to). Is he sending calendar year or fiscal year from the front end?